traffic_ops/app/t/api/1.2/config_files_apache_traffic_server.t

```diff
@@ -72,7 +72,7 @@ my $count_response = sub {
 	return $t->success( is( scalar(@$r), $count ) );
 };
 
-$t->get_ok('/api/1.2/deliveryservices/list?logsEnabled=true')->status_is(200)->$count_response(2)
+$t->get_ok('/api/1.2/deliveryservices/list?logsEnabled=true')->status_is(200)->$count_response(3)
```

Comment:
this doesn't seem related to your PR. you might want to rebase or merge from master and see if this change is still necessary.